content/how-to/bpmn/tune-performance.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Tune BPMN performance'
+date: '2024-02-09T09:47:47-03:00'
+categories: ["how-to"]
+description: Tips debug and improve the performance of your BPMN process
+weight: 300
+menu:
+  main:
+    parent: howto-bpmn
+    identifier:
+    name: Tune performance
+---
+
+This page documents some tips to debug [BPMN workflows]({{< relref "/how-to/bpmn/create-workflow" >}}) and improve their performance.
+
+Manufacturing events can generate a vast amount of data.
+Rhize has no limits on the data it can ingest,
+and a BPMN workflow can have any number of logical flows and data transformations.
+So an inefficient BPMN process can introduce performance degradations.
+
+When writing and debugging a BPMN workflow, consider these tips.
+
+## Avoid parallel gateways
+
+Running processes in [parallel]({{< relref "/how-to/bpmn/bpmn-elements#parallel-gateway" >}}) can increase the workflow's complexity by an order of magnitude.
```

Comment:
@tomhollingworth Do you have some advice on how I can get more specific here? What would the main effects be? Over taxed CPU? Out of memory?

Reply:
Increased memory usage of NATS